### ISSUE TITLE
Maintenance help needed?

### ISSUE THREAD

**Issue by lprimak** [jbake-org/jbake#785]
Hi,

I can see that JBake has not been maintained for more than a year.
Is some help with maintenance needed?
What is the future of JBake? I would like for it to get some more love.

**Comment by lprimak** [comment]
@jonbullock Any insight would be appreciated, thank you!

**Comment by ottlinger** [comment]
Does anyone know if Jon is still around - all his webpages seem not to work anymore ..... is it only him with admin access to the jbake org?

* https://bullock.io/
* https://jonbullock.co.uk/2013/05/jbake-v2-1-released.html

ping @jonbullock

**Comment by jonbullock** [comment]
I'm still around, and I do want to continue JBake, however my circumstances have change which is limiting my involvement. However I hope this will be resolved soon.

**Comment by lprimak** [comment]
@jonbullock Perhaps it's time to add maintainers or hand the project to someone who is willing to maintain it?
Sounds like @ottlinger or me are volunteering. It's been 6 months since your last comment and 3 years since any release of JBake...

Thank you for your understanding. I know you are not obligated to do anything.

**Comment by ancho** [comment]
Hey. Sorry. I've been ignoring this project for quite some time now.
There are various reasons for this. 
And it's been so long since I actively worked on the project that I don't even remember some of them anymore.

One reason is that I hardly use jbake privately anymore, or only indirectly via doctoolchain.
And, of course, the struggles of every working person with a life. Lack of time and other hobbies.

But I had a lot of fun with it in the past and still consider it a practical tool.
I would find it a shame if the project came to a standstill.

That's at least one of the reasons why I'm currently becoming a little more active again.

I'm currently trying to find out what I can actually do.

I'll try to support as much as I can until the future of the project is clarified.

What I can say so far.

Jon has the main permissions for the project.
Only he can make changes to the organization and the repositories.

But @manikmagar  and I can manage issues and merge PRs.

The deployment of the homepage should be fully automated. (If everything still works there. I didn't look into it)
If I remember correctly that is happening at gitlab currently.

As far as I can see, GitHub releases and SDK publishing are managed via jreleaser using GitHub actions.
The Maven publishing of artifacts has been done manually so far via gradle maven-publish plugin.

I obviously have an account at central.sonatype.com and can see the org.jbake namespace.
This means that, in theory, publishing the artifacts and plugins will also be possible in the future.

That's all I know so far.

But let's see if we can dust off the project first, I would say.

I started with an upgrade of dependencies. 
See https://github.com/jbake-org/jbake/pull/790

Btw. if someone is developing on a windows machine, or can figure out a solution for the failing AssetTest under Windows.
Help there is highly appreciated. 

See https://github.com/jbake-org/jbake/actions/runs/19581019774/job/56078659812?pr=790#step:5:497

**Comment by manikmagar** [comment]
I echo almost everything that @ancho said in previous comment. I too haven't used JBake in long time and priorities had shifted. Happy to help to keep this project active.

@ancho I had setup that Gitlab pipeline for publishing home page but that was long time back. But that was switched to [GitHub pages](https://github.com/jbake-org/jbake.org/pull/156) later. So latest should be from GitHub pages only.

Looks like the jreleaser process is outdated/broken. We can look into that as well and fix it.

I may not be super active like in the past due to other commitments but at least I will make sure to unblock everyone's efforts with the access we have.

**Comment by jonbullock** [comment]
@lprimak  The blocker for me is nearly resolved (I'm talking days not weeks or months) but I'm reluctant to discuss further on a public forum. 

As you can see there are other committers and there seems to be rejuvenated push for the project to move forward which is great to see.

**Comment by lprimak** [comment]
Thank you Jon for responding.
I am glad that your priorities may align with JBake in the future.

However, the original question is still remains unanswered. Both Manik and Frank don't use JBake anymore,
so their interests do not directly align with the project. It's hardly expect them to do any work simply due to that fact.

Are you willing to add committers to the project that actually use and depend on it today?

**Comment by brianoliver** [comment]
It's with a very heavy heart that I suggest this, but I'm wondering if the community should just fork this project, simply so we can continue the active maintenance and improvements of it?   Looking at the numbers, there's over 300 forks of it already.  There's over 1000 stars!  There's 22 pretty good pending PRs and over 145 issues to review.    There's literally a community waiting to dive in an help, but we can't :( 

I know open source projects take a lot of effort, sometimes an insane amount, but when it seems there are a number of people that want to contribute and we can't, that's truly sad.   @jonbullock ... let us help you!  Please!

I've personally used and supported the use of JBake for over 8 years.   It's wonderful and I would like to see it grow.  So much so, I'll put my own time and money into it.

The challenge for me is that I now have a number of new open source projects to release, and documenting them with JBake would be ideal.   However I can't really recommend that for an enterprise when JBake is sort of a zombie project.

**Comment by lprimak** [comment]
I would agree with the above. JBake needs fresh blood. None of the current committers are working on it, as clearly evidenced by the lack of progress, despite statements to the contrary.
Issues are still not getting resolved, PRs not getting merged, etc. etc.

**Comment by ottlinger** [comment]
Another 4 months is over and the project seems to be in hibernate-mode - any updates or plans for a revival? ping @jonbullock

**Comment by lprimak** [comment]
Is BakeToTheFuture fork maintained? If it is, maybe we should join that effort. At this point I believe it's safe to say that JBake will not be revived and a fork needs to be made or existing fork should be joined. 

I also do not believe at this point that @jonbullock is willing to give up control of the project nor is he willing to add maintainers.

It's just been too long and facts prove the above.
